### ISSUE TITLE
Doesn't work for large bundles

### ISSUE THREAD

**Issue by jb55** [matthewbauer/nix-bundle#20]
Trying to bundle tensorflow as a test:

    nix-bundle python27Packages.tensorflow bin/tensorboard

generates a 1.5GB archive. When I try to run it, it runs out of memory.

I'm guessing nix-bundle isn't a good solution for these scenarios? Perhaps it's an issue with arx?

**Comment by probonopd** [comment]
@jb55 at least with AppImages, this should not be a problem, since we are just loop-mounting the filesystem image. Have used games in the order of gigabytes w/o issues.

**Comment by jb55** [comment]
@probonopd yeah I figured that was the issue, for large bundles loop-mounting seems to be the way to go. I would say we could try to do something like that with nix-bundle, but it might be easier to just generate an AppImage once that's working...

**Comment by probonopd** [comment]
nix-bundle has a script that can make AppImages already...

**Comment by jb55** [comment]
yeah I just assumed it wasn't working because it doesn't seem to work for me... hmm...

**Comment by probonopd** [comment]
It has open issues, yes... but I don't see why they couldn't be fixed.
